satpy/tests/writer_tests/test_cf.py

```diff
@@ -664,6 +664,48 @@ def test_make_alt_coords_unique(self):
         self.assertNotIn('var1_acq_time', res['var1'].coords)
         self.assertNotIn('var2_acq_time', res['var2'].coords)
 
+    def test_dataset_is_projection_coords(self):
```

Comment:
Can you make a separate test case for these two test methods? they have a lot of test data in common which you can then refactor into the `setUp` method of the test case